Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -151,15 +151,17 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
             };
         }
         public override Task Shutdown() {
-            Interlocked.Exchange(ref _analyzer, null)?.Dispose();
-            _projectFiles.Clear();
+            Analyzer?.Dispose();
```

Comment:
Is `Analyzer` protected against double dispose?

Reply:
Nope

Reply:
However, this is single thread context